src/EnergyPlus/CondenserLoopTowers.hh

```diff
@@ -216,7 +254,7 @@ namespace CondenserLoopTowers {
 
         // From module level variables, apparently the module AirFlowRateRatio was used slightly different from the struct's AirFlowRatio variable
         //  so removing this caused diffs that I did not spend time investigating...they might be fine diffs, check some time later
-        Real64 __AirFlowRateRatio;
+        Real64 airFlowRateRatio;
```

Comment:
Renamed some variables. CLion is complaining because leading underscores are supposed to indicate a protected identifier.